specs/experimental/security-council-safe.md

```diff
@@ -23,7 +23,48 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Liveness checking Mechanism
+The Security Council uses a specially extended Safe multisig contract to provide additional security
+guarantees on top of those provided by the Safe contract.
+
+## Deputy guardian module
+
+The Security Council acts as the Guardian, which is authorized to activate the [Superchain Pause](../protocol/superchain-configuration.md#pausability) functionality and for [blacklisting](../experimental/fault-proof/stage-one/bond-incentives.md#authenticated-roles) dispute game contracts.
+
+However the Security Council cannot be expected to react quickly in an emergency situation.
+Therefore the Pausability Pass Through module enables the Security Council to delegate share this
```

Comment:
Typo "delegate share"—does it delegate the authority or share it?

Reply:
Yes, I went back and forth between them. 

I think 'share' is more accurate though, as both entities will be able to make the call.